docs/benchmarks/fio.md

```diff
@@ -25,9 +25,9 @@ You can find [configuration examples](https://github.com/xridge/kubestone/tree/m
 
 
 ## Sample benchmark
-To run a sample benchmark with PVC mode, the following command can be used:
+To run a sample fio benchmark, the following command can be used:
 ```bash
-$ kustomize build github.com/xridge/kubestone/config/samples/fio/overlays/pvc | kubectl create --namespace kubestone -f -
+$ kustomize build github.com/xridge/kubestone/config/samples/fio/base | kubectl create --namespace kubestone -f -
```

Comment:
According to kustomize documentation you don't need to specify the 'base', e.g. this would also work:
`kustomize build github.com/xridge/kubestone/config/samples/fio | kubectl create --namespace kubestone -f - `